### PR TITLE
Update docker registry note

### PR DESCRIPTION
**Pull request by pavera**
### Why:

Closes [issue link]

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Updating dependabot configuration documentation to indicate our positively supported docker container registries

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @pavera Just checking in, I noticed this is a draft PR. Let me know when it's ready for writer review! ✨

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 1]
```diff
@@ -778,7 +778,11 @@ registries:
 
 {% note %}
 
-**Note:** We don't support the Azure Container Registry (ACR).
+**Note:** We support the container registries listed here:
```
Comment (jeffwidman) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:781]:
```suggestion
**Note:** We currently support the container registries listed here:
```

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 2]
```diff
@@ -778,7 +778,11 @@ registries:
...
+* Docker Hub
+* GitHub Container Registry
+* GCR (Google Cloud)
+* Private ECR (AWS)
```
Comment (jeffwidman) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:785]:
```suggestion
* Private ECR (AWS) - public ECR support is tracked [here](https://github.com/dependabot/dependabot-core/issues/4212).
```

**Review by jeffwidman — COMMENTED** [review]
Somewhere in this doc there's a mention that we don't support Azure ACR... could you add also add a hyperlink in that mention to https://github.com/dependabot/dependabot-core/issues/3689? 

That way users that are interested can hopefully clickthrough to see if anything's changed, and also they'll see your latest comment that support ACR is blocked by their non-standard APIs rather than our lack of interest.

**Comment by cmwilson21** [comment]
👋 @pavera, thanks for marking this as ready! I'll get it triaged for review ⚡

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 3]
```diff
@@ -778,7 +778,11 @@ registries:
 
 {% note %}
```
Comment (jeffwidman) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:780]: I would maybe start with a generic statement like:

> Dependabot works with any container registry that implements [the OCI container registry spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md). For private registries we support authentication via [a central service](https://docs.docker.com/registry/spec/auth/token/) or via [Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). 

And then your note of specific registries can be restated as:
> Registries that are currently known to work with Dependabot include:
> * Docker Hub
>  * GitHub Container Registry
> ...

That way it shifts the conversation from "Dependabot doesn't support X registry" to "X registry doesn't support the OCI spec, so please go talk to them"...

**Review by jeffwidman — COMMENTED** [review]
Had a few comments re: potential ways to clarify things... feel free to ping me on slack to discuss or if you want to pair on wordsmithing...

**Comment by mchammer01** [comment]
@pavera @jeffwidman 👋🏻 - happy to improve this note if you think it's not clear enough and could lead to Support tickets.

It would be helpful if you could agree on what you'd like to convey to our customers as you are the experts, and I'll review your edits on behalf of the docs team once these are ready for me.

We try to avoid using the passive voice in our docs (as per our content model) so I may tweak your updates at review time.

Feel free to ping me when this is ready for me. Thank you 🙇🏻‍♀️ 🙂

**Comment by pavera** [comment]
@mchammer01 - I believe this is now ready for your review thanks for your patience :)

**Review by jeffwidman — DISMISSED** [review]

**Review by mchammer01 — COMMENTED** [review]
This LGTM ✨ 
Just made a few editorial suggestions in line with our content model, that I'm going to apply.

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 4]
```diff
@@ -776,11 +776,15 @@ registries:
 
 ### `docker-registry`
 
-{% note %}
+{% data variables.product.prodname_dependabot %} works with container registries that implement the OCI container registry. For more information, see [https://github.com/opencontainers/distribution-spec/blob/main/spec.md](https://github.com/opencontainers/distribution-spec/blob/main/spec.md).  {% data variables.product.prodname_dependabot %} supports authentication to private registries via a central service. For further details, see [https://docs.docker.com/registry/spec/auth/token/](https://docs.docker.com/registry/spec/auth/token/).
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:779]:
```suggestion
{% data variables.product.prodname_dependabot %} works with container registries that implement the OCI container registry. For more information, see [https://github.com/opencontainers/distribution-spec/blob/main/spec.md](https://github.com/opencontainers/distribution-spec/blob/main/spec.md).  {% data variables.product.prodname_dependabot %} supports authentication to private registries via a central service. For further details, see [Token Authentication Specification](https://docs.docker.com/registry/spec/auth/token/) in the Docker documentation.
```

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 5]
```diff
@@ -776,11 +776,15 @@ registries:
...
 
-**Note:** We don't support the Azure Container Registry (ACR).
+We currently support the container registries listed here:
+
+* Docker Hub
+* {% data variables.product.company_short %} {% data variables.product.prodname_container_registry %}
+* GCR (Google Cloud)
+* Private ECR (AWS) - public ECR support is tracked [here](https://github.com/dependabot/dependabot-core/issues/4212).
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:786]: Removing inline link
```suggestion
* Private ECR (AWS) - public ECR support is tracked in [https://github.com/dependabot/dependabot-core/issues/4212](https://github.com/dependabot/dependabot-core/issues/4212).
```

**Review by mchammer01 — COMMENTED** [review]
Further improving a couple of links.

**Review by mchammer01 — APPROVED** [review]
:shipit:

**Pull request merged**